examples/EXAMPLES_LICENSE

```diff
@@ -0,0 +1,14 @@
+Zero-Clause BSD License
+
+Copyright (c) 2021 Toitware
```

Comment:
Hmm. This is (c) rather than (C) and Toitware rather than Toitware ApS. Have you thought about how to make it even more consistent?